docs/en/kubernetes/Metrics-On-Kubernetes.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: global
+title: Metrics System On Kubernetes
+nickname: Metrics
+group: Kubernetes
+priority: 2
+---
+
+This documentation focus on how to get metrics from Alluxio deployed on Kubernetes.
```

Comment:
Can the summary be
- one line about the usage of metrics, refer to link
- one line about the popularity of deployment Alluxio on kubernetes, refer to link
- and This documentation focus on how to get metrics from Alluxio deployed on Kubernetes.